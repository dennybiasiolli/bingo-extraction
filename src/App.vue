<template>
  <v-app>
    <AppBar />
    <v-main>
      <Home />
    </v-main>
    <Footer />
  </v-app>
</template>

<script>
import AppBar from '@/views/AppBar.vue';
import Footer from '@/views/Footer.vue';
import Home from '@/views/Home.vue';

export default {
  name: 'App',

  components: {
    AppBar,
    Footer,
    Home,
  },

  data: () => ({
    //
  }),
};
</script>
