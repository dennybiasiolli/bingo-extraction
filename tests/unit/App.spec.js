import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App.vue', () => {
  it('renders as expected', () => {
    const wrapper = shallowMount(App, {
      stubs: ['v-app', 'v-app-bar', 'v-img', 'v-spacer', 'v-btn', 'v-icon', 'v-main'],
    });
    expect(wrapper).toMatchSnapshot();
  });
});
