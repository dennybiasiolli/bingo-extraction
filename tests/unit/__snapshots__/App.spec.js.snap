// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders as expected 1`] = `
<v-app-stub>
  <v-app-bar-stub app="" color="primary" dark="">
    <div class="d-flex align-center">
      <v-img-stub alt="Vuetify Logo" contain="" src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png" transition="scale-transition" width="40" class="shrink mr-2"></v-img-stub>
      <v-img-stub alt="Vuetify Name" contain="" min-width="100" src="https://cdn.vuetifyjs.com/images/logos/vuetify-name-dark.png" width="100" class="shrink mt-1 hidden-sm-and-down"></v-img-stub>
    </div>
    <v-spacer-stub></v-spacer-stub>
    <v-btn-stub href="https://github.com/vuetifyjs/vuetify/releases/latest" target="_blank" text=""><span class="mr-2">Latest Release</span>
      <v-icon-stub>mdi-open-in-new</v-icon-stub>
    </v-btn-stub>
  </v-app-bar-stub>
  <v-main-stub>
    <home-stub></home-stub>
  </v-main-stub>
</v-app-stub>
`;
